### ISSUE TITLE
Tiny Nerf Colab Evaluation Bug

### ISSUE THREAD

**Issue by johnmaxh** [krrish94/nerf-pytorch#36]
It looks like when evaluating the loss in the last cell of the colab, the code uses
`loss = torch.nn.functional.mse_loss(rgb_predicted, target_img)`

But believe this should actually be
`loss = torch.nn.functional.mse_loss(rgb_predicted, testimg)`

Since we use `testpose` for `rgb_predicted` we should compare to `testimg` not `target_img`, which is just a random training sample.
